### PR TITLE
Fix zpaq

### PR DESCRIPTION
**Pull request by demhademha**
Fix zpaq on iOS

### CODE REVIEW

File: zpaq.mk [hunk 1]
```diff
@@ -5,31 +5,40 @@ endif
 SUBPROJECTS  += zpaq
 ZPAQ_VERSION := 7.15
 DEB_ZPAQ_V   ?= $(ZPAQ_VERSION)
+ZPAQ_CPPFLAGS= -Dunix
 
 zpaq-setup: setup
 	$(call GITHUB_ARCHIVE,zpaq,zpaq,$(ZPAQ_VERSION),$(ZPAQ_VERSION))
 	$(call EXTRACT_TAR,zpaq-$(ZPAQ_VERSION).tar.gz,zpaq-$(ZPAQ_VERSION),zpaq)
 
+ifneq ($(MEMO_ARCH),amd64)
```
Comment (Diatrus) [inline zpaq.mk:14]: I don’t know that there *is* an amd64 MEMO_ARCH

File: zpaq.mk [hunk 2]
```diff
@@ -5,31 +5,40 @@ endif
...
 ifneq ($(wildcard $(BUILD_WORK)/zpaq/.build_complete),)
 zpaq:
 	@echo "Using previously built zpaq."
 else
 zpaq: zpaq-setup
 	+$(MAKE) -C $(BUILD_WORK)/zpaq \
-		CXX=$(CXX) \
-		CXXFLAGS="$(CXXFLAGS)" \
-		LDFLAGS="$(LDFLAGS)"
+	CXX=$(CXX) \
```
Comment (Diatrus) [inline zpaq.mk:24]: Indentation issues

File: zpaq.mk [hunk 3]
```diff
@@ -5,31 +5,40 @@ endif
...
+	PREFIX=$(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
 	touch $(BUILD_WORK)/zpaq/.build_complete
 endif
+
 zpaq-package: zpaq-stage
+
 	# zpaq.mk Package Structure
 	rm -rf $(BUILD_DIST)/zpaq
 
 	# zpaq.mk Prep zpaq
```
Comment (Diatrus) [inline zpaq.mk:40]: Unnecessary whitespace added

File: zpaq.mk [hunk 4]
```diff
@@ -39,6 +48,7 @@ zpaq-package: zpaq-stage
 	$(call PACK,zpaq,DEB_ZPAQ_V)
 
 	# zpaq.mk Build cleanup
+
 	rm -rf $(BUILD_DIST)/zpaq
 
-.PHONY: zpaq zpaq-package
+	.PHONY: zpaq zpaq-package
```
Comment (Diatrus) [inline zpaq.mk:54]: Indentation again

**Comment by demhademha** [comment]
Should be good now

**Comment by nasrullah1212** [comment]
د
